mimic/model/nova_objects.py

```diff
@@ -499,7 +500,7 @@ def request_read(self, http_get_request, server_id, absolutize_url):
         server = self.server_by_id(server_id)
         if server is None:
             http_get_request.setResponseCode(404)
-            return None
+            return dumps(not_found_response('servers'))
```

Comment:
Good point, and maybe this should be split up by service anyway.  I'll file an issue.  :)